apps/storybook/src/DomainControls.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import { DomainControls } from '@h5web/lib';
+import type { Domain } from '@h5web/shared';
+import { useToggle } from '@react-hookz/web';
+import type { Meta, StoryObj } from '@storybook/react';
+import { useEffect, useState } from 'react';
+
+const meta = {
+  title: 'Toolbar/DomainControls',
+  component: DomainControls,
+  argTypes: {
+    sliderDomain: { control: false },
+    errors: { control: false },
+    isAutoMin: { control: false },
+    isAutoMax: { control: false },
+    isEditingMin: { control: false },
+    isEditingMax: { control: false },
+  },
+} satisfies Meta<typeof DomainControls>;
+
+export default meta;
+type Story = StoryObj<typeof DomainControls>;
+
+export const Default = {
+  render: (args) => {
+    const { dataDomain } = args;
+
+    const [sliderDomain, setSliderDomain] = useState<Domain>(dataDomain);
+    const [isAutoMin, toggleAutoMin] = useToggle();
+    const [isAutoMax, toggleAutoMax] = useToggle();
+    const [isEditingMin, toggleEditingMin] = useToggle();
+    const [isEditingMax, toggleEditingMax] = useToggle();
+
+    useEffect(() => {
+      setSliderDomain(dataDomain);
+    }, [dataDomain]);
+
+    return (
+      <div style={{ maxWidth: '18rem' }}>
+        <DomainControls
+          sliderDomain={sliderDomain}
+          dataDomain={dataDomain}
+          errors={{}}
```

Comment:
I'm not bothering with the errors, since this would involve managing a `customDomain` and computing `visDomain` and `safeVisDomain` (which requires knowing the `scaleType`).

I decided to keep things simple and manage only a simple `sliderDomain` state.